tests/test_conversion_base.py

```diff
@@ -1316,6 +1317,32 @@ def test_convert_value_regex_unbound_not_escaped_escape(test_backend):
     )
 
 
+def test_convert_value_regex_value_list():
+    pipeline = ProcessingPipeline(
+        [ProcessingItem(ValueListPlaceholderTransformation(["test"]))],
+        vars={"test": ["pat.*tern/foobar", "pat.*te\\rn/foobar"]},
+    )
+    backend = TextQueryTestBackend(pipeline)
+    assert (
+        backend.convert(
+            SigmaCollection.from_yaml(
+                """
+            title: Test
+            status: test
+            logsource:
+                category: test_category
+                product: test_product
+            detection:
+                sel:
+                    field|re|expand: "%test%"
```

Comment:
Please also add a test where modifiers are applied after expansion. Currently possibly problematic usage of `SigmaString.original` isn't covered.